pkg/cmd/cli/describe/projectstatus.go

```diff
@@ -607,6 +607,11 @@ type serviceLoader struct {
 	items     []kapi.Service
 }
 
+// NewServiceLoader returns a new serviceLoader
+func NewServiceLoader(namespace string, lister kclient.ServicesNamespacer) *serviceLoader {
```

Comment:
Define Java like?  "New*" something is canonical go for controlling setting private fields and options you don't want someone to use.  However, it should be used judiciously - it's only useful when you're exporting or have a large package.